.github/workflows/ci-python-sdk.yaml

```diff
@@ -23,6 +23,7 @@ defaults:
     working-directory: python-sdk
 jobs:
   Markdown-link-check:
+    if: github.event.action != 'labeled'
```

Comment:
We don't want to run this jobs when a label is added as it is redundant with when PR is created